nixos/lib/make-system-tarball.sh

```diff
@@ -52,9 +52,10 @@ $extraCommands
 
 mkdir -p $out/tarball
 
-tar cvJf $out/tarball/$fileName.tar.xz * $extraArgs
+rm env-vars
+
+tar --sort=name --mtime='1970-01-01' -cvJf $out/tarball/$fileName.tar.xz * $extraArgs
```

Comment:
IIRC tar needs more magic options or it will capture the name of the build user: 3d509896511fac1c73307fedf2060c9dddebf73a